themes/aframe/layout/partials/blog/metadata.ejs

```diff
@@ -0,0 +1,6 @@
+<div class="post__metadata">
+  <p class="post__timestamp"><%- blog_date(post.date) %></p>
+  <% if (post.author) { %>
+    <p class="post__author">|<%- blog_attribution(post.author) %></p>
```

Comment:
this is pretty awk. can we do this with CSS?  `.post__author:before { content: "|"; }`